src/actions/guides/tutorial/02_assocations.cr

```diff
@@ -85,12 +85,12 @@ class Guides::Tutorial::Associations < GuideAction
 
     puts "Truncating the Fortunes table"
 
-    FortuneQuery.truncate
+    FortuneQuery.truncate(cascade: true)
 
     puts "done!"
     ```
 
-    Once added, save and exit the file. This will tell Lucky to compile the code, and execute it, which will run a `TRUNCATE` on
+    Once added, save and exit the file. This will tell Lucky to compile the code, and execute it, which will run a [`TRUNCATE`](/guides/database/deleting-records#truncate-table) on
```

Comment:
```suggestion
    Once added, save and exit the file. This will tell Lucky to compile the code, and execute it, which will run a [`TRUNCATE`](#{Guides::Database::DeletingRecords.path}) on
```

Reply:
Goot know. I was wondering about that.